link.js

```diff
@@ -14,6 +14,7 @@ H5P.Link = (function ($) {
     // Add default parameters
     parameters = $.extend(true, {
       title: 'New link',
+      openInTheSameTab: false,
```

Comment:
The default should be true.

In the Jira story the link text was "Open the link in new window/tab" with unchecked checkbox (false)

I believe it is more natural (for the way we use tapbyl) to make "open in a new tab" the exception and therefore something you have to explicitely enable.

Reply:
Fixed